packages/deprecation-crawler/src/lib/output-formatters/markdown/generate-group-based-formatter.ts

```diff
@@ -102,31 +107,23 @@ the following version specifier are set for the rxjs dependencies in StackBlitz:
 `;
 }
 
-
-export async function getRemoteUrl(): Promise<string> {
```

Comment:
Moved this to the `utils` folder because the remote Url is now added to the crawled deprecations during the checkout step.